com.inklestudios.Inky.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```

Comment:
Ideally this, appdata and icon files should be upstreamed.

Reply:
Indeed. I will open a PR for them.

Reply:
@barthalion this is the desktop file they have upstream: https://github.com/inkle/inky/blob/master/resources/com.inkle.inky.desktop . It is for AppImage. Do you think there is a way to merge this one without affecting AppImage?

1. Is the shebang `#!/usr/bin/env xdg-open` mandatory in appimage?
2. Maybe I can use `desktop-file-edit` in the manifest to change `Exec` and `Path`?
3. It seems they have the app ID wrong, the studio's domain is clearly `inklestudios.com`, not `inkle.com`.

Reply:
1. No. It's probably a hack to run a desktop file from CLI after `chmod +x`.
2. Yeah, that's what maintainers usually do.
3. Apparently so! Feel free to fix that after application is merged, it's not a blocker.